src/cookbooks/bigquery/access.md

```diff
@@ -0,0 +1,147 @@
+## Table of Contents
+<!-- toc -->
+
+# Access
+
+Although multiple methods are available to get access to BigQuery, you may want to select the primary interface, which uses [re:dash](https://sql.telemetry.mozilla.org/). You can use any of the following interfaces. However, all other interfaces require access to be provisioned.
+
```

Comment:
This section doesn't seem very useful to me: with the exception of one important note, it just repeats content which lies immediately below. Let's take it out and move the meaningful caveat about bigquery access to the next section.

```suggestion
```